PlateSpace/Web/README.md

```diff
@@ -278,60 +228,13 @@ Stitch lets you set up rules for every collection in your database, for instance
   * __Write__:
   ```json
   {
-  "%and":
-    [
-      {
-        "%%root.owner_id": "%%user.id"
-      },
-      {
-        "%or": [
-          {
-            "%and": [
-              {
-                "%%true": {
-                  "%pipeline": {
-                    "name": "userHasSingleReview",
-                    "args": {
-                      "userId": "%%user.id",
-                      "restaurantId": "%%root.restaurantId"
-                    }
-                  }
-                }
-              },
-              {
-                "%%prevRoot": {
-                  "%exists": true
-                }
-              }
-            ]
-          },
-          {
-            "%and": [
-              {
-                "%%false": {
-                  "%pipeline": {
-                    "name": "userHasSingleReview",
-                    "args": {
-                      "userId": "%%user.id",
-                      "restaurantId": "%%root.restaurantId"
-                    }
-                  }
-                }
-              },
-              {
-                "%%prevRoot": {
-                  "%exists": false
-                }
-              }
-            ]
-          }
-        ]
-      }
-    ]
+  "owner_id": "%%user.id"
   }
   ```
 * Leave the `owner_id` field as is.
 
+__Important note__: There is a unique compound index on the reviewsRatings collection (restaurant_id/user_id) to prevent users from adding more than one review per restaurant.
```

Comment:
@rlondner  -- double checking -- we state in the schema for reviewsRatings collection that the fields are restaurantId and user_id.

Should we 1) change to both use underscore?  2) if not, we should double check that we are creating the compound index on the correct fields.

Other than that, I'm good with this .